### PR TITLE
[IMP] project: improve task count and label in project updates

### PR DESCRIPTION
**Pull request by padh-odoo**
With this commit, we are displaying 0 and the task label (Name of the tasks)
if the number of tasks recorded is equal to 0 in the project updates.
Additionally, we are showing the same task label in the stat button on the
right side panel and smart button of task on the project.

task-3893332

### CODE REVIEW

File: addons/project/views/project_project_views.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
                     <div class="oe_button_box" name="button_box" groups="base.group_user">
                         <button class="oe_stat_button" type="object" name="action_view_tasks" icon="fa-check">
                             <div class="o_field_widget o_stat_info">
-                                <span class="o_stat_text">Tasks</span>
+                                <field name="label_tasks" readonly="1"/>
```
Comment (ppr-odoo) [inline addons/project/views/project_project_views.xml:46]:
```suggestion
                                <field name="label_tasks">
```

A related field is default readonly, so we don't need to specify the readonly attribute.

**Review by damrOdoo — COMMENTED** [review]
Lgtm. Thanks for your imp.

**Comment by xavierbol** [comment]
robodoo r+

File: addons/hr_timesheet/data/hr_timesheet_demo.xml [hunk 2]
```diff
@@ -3796,12 +3796,29 @@
         <field name="user_id" eval="ref('base.user_demo')"/>
         <field name="progress" eval="30"/>
         <field name="status">on_hold</field>
+        <field name="uom_id" ref="uom.product_uom_hour"/>
```
Comment (xavierbol) [inline addons/hr_timesheet/data/hr_timesheet_demo.xml:3799]: Actually, it is really needed? 

It seems we add it in the create method. 🤔

File: addons/hr_timesheet/data/hr_timesheet_demo.xml [hunk 3]
```diff
@@ -3796,12 +3796,29 @@
...
     </record>
     <record id="project_update_2" model="project.update" context="{'default_project_id': ref('project.project_project_2')}">
         <field name="name">Weekly review</field>
         <field name="user_id" eval="ref('base.user_admin')"/>
         <field name="progress" eval="30"/>
         <field name="status">off_track</field>
+        <field name="uom_id" ref="uom.product_uom_hour"/>
```
Comment (xavierbol) [inline addons/hr_timesheet/data/hr_timesheet_demo.xml:3806]: same comment here

**Review by xavierbol — COMMENTED** [review]
robodoo r-

Thanks for your work, I left 2 comments.

**Review by xavierbol — COMMENTED** [review]
robodoo r+

**Comment by xavierbol** [comment]
robodoo rebase-ff

**Pull request closed without merging**